internal/server/turn.go

```diff
@@ -17,7 +18,37 @@ func (s *SOMASServer) runTurn() error {
 		return err
 	}
 
-	if err := s.runOrgs(); err != nil {
+	allActions := []common.Action{}
+
+	// run all orgs and get all actions
+	if actions, err := s.runOrgs(); err != nil {
+		return err
+	} else {
+		allActions = append(allActions, actions...)
+	}
+
+	// get all end of turn actions
+	if actions, err := s.getEndOfTurnActions(); err != nil {
+		return err
+	} else {
+		allActions = append(allActions, actions...)
+	}
+
+	// !!!!!!!!!!!!

```

Comment:
Leaving this here for now--to be removed when we have actual actions implemented by other infra reps.